diff --git a/node_modules/nodejs-mobile-react-native/react-native.config.js b/node_modules/nodejs-mobile-react-native/react-native.config.js
index 857c697..97ed7cd 100644
--- a/node_modules/nodejs-mobile-react-native/react-native.config.js
+++ b/node_modules/nodejs-mobile-react-native/react-native.config.js
@@ -5,7 +5,7 @@ module.exports = {
         sourceDir: './android'
       },
       ios: {
-        project: './ios/RNNodeJsMobile.xcodeproj',
+        project: './apps/mobile/ios/MnemeMobile.xcodeproj',
         scriptPhases: [
           {
             name: '[NODEJS MOBILE] Copy Node.js Project files',
diff --git a/node_modules/nodejs-mobile-react-native/scripts/ios-build-native-modules.sh b/node_modules/nodejs-mobile-react-native/scripts/ios-build-native-modules.sh
index d0d6f7c..4323c57 100644
--- a/node_modules/nodejs-mobile-react-native/scripts/ios-build-native-modules.sh
+++ b/node_modules/nodejs-mobile-react-native/scripts/ios-build-native-modules.sh
@@ -88,7 +88,7 @@ done
 
 # Apply patches to the modules package.json
 if [ -d "$NODEPROJ"/node_modules/ ]; then
-  PATCH_SCRIPT_DIR="$( cd "$PROJECT_DIR" && cd ../node_modules/nodejs-mobile-react-native/scripts/ && pwd )"
+  PATCH_SCRIPT_DIR="$( cd "$PROJECT_DIR" && cd ../../../node_modules/nodejs-mobile-react-native/scripts/ && pwd )"
   NODEJS_PROJECT_MODULES_DIR="$( cd "$NODEPROJ" && cd node_modules && pwd )"
   node "$PATCH_SCRIPT_DIR"/patch-package.js $NODEJS_PROJECT_MODULES_DIR
 fi
@@ -102,7 +102,7 @@ if [ -f ~/.cargo/env ]; then
 fi
 
 # Rebuild modules with right environment
-NODEJS_HEADERS_DIR="$( cd "$PROJECT_DIR" && cd ../node_modules/nodejs-mobile-react-native/ios/libnode/ && pwd )"
+NODEJS_HEADERS_DIR="$( cd "$PROJECT_DIR" && cd ../../../node_modules/nodejs-mobile-react-native/ios/libnode/ && pwd )"
 pushd $NODEPROJ
 if [ "$PLATFORM_NAME" == "iphoneos" ]
 then
diff --git a/node_modules/nodejs-mobile-react-native/scripts/ios-copy-nodejs-project.sh b/node_modules/nodejs-mobile-react-native/scripts/ios-copy-nodejs-project.sh
index aefe0e0..d29fe6e 100644
--- a/node_modules/nodejs-mobile-react-native/scripts/ios-copy-nodejs-project.sh
+++ b/node_modules/nodejs-mobile-react-native/scripts/ios-copy-nodejs-project.sh
@@ -2,7 +2,7 @@
 set -e
 
 NODEJS_ASSETS_DIR="$( cd "$PROJECT_DIR" && cd ../nodejs-assets/ && pwd )"
-NODEJS_BUILT_IN_MODULES_DIR="$( cd "$PROJECT_DIR" && cd ../node_modules/nodejs-mobile-react-native/install/resources/nodejs-modules/ && pwd )"
+NODEJS_BUILT_IN_MODULES_DIR="$( cd "$PROJECT_DIR" && cd ../../../node_modules/nodejs-mobile-react-native/install/resources/nodejs-modules/ && pwd )"
 
 if [ -d "$CODESIGNING_FOLDER_PATH/nodejs-project/" ]; then
   rm -rf "$CODESIGNING_FOLDER_PATH/nodejs-project/"
diff --git a/node_modules/nodejs-mobile-react-native/scripts/ios-sign-native-modules.sh b/node_modules/nodejs-mobile-react-native/scripts/ios-sign-native-modules.sh
index 901b6c1..6688314 100644
--- a/node_modules/nodejs-mobile-react-native/scripts/ios-sign-native-modules.sh
+++ b/node_modules/nodejs-mobile-react-native/scripts/ios-sign-native-modules.sh
@@ -45,7 +45,7 @@ find "$CODESIGNING_FOLDER_PATH/nodejs-project/" -name "*.o" -type f -delete
 find "$CODESIGNING_FOLDER_PATH/nodejs-project/" -name "*.a" -type f -delete
 
 # Create Info.plist for each framework built and loader override.
-PATCH_SCRIPT_DIR="$( cd "$PROJECT_DIR" && cd ../node_modules/nodejs-mobile-react-native/scripts/ && pwd )"
+PATCH_SCRIPT_DIR="$( cd "$PROJECT_DIR" && cd ../../../node_modules/nodejs-mobile-react-native/scripts/ && pwd )"
 NODEJS_PROJECT_DIR="$( cd "$CODESIGNING_FOLDER_PATH" && cd nodejs-project/ && pwd )"
 node "$PATCH_SCRIPT_DIR"/ios-create-plists-and-dlopen-override.js $NODEJS_PROJECT_DIR
 
